docker/sonarqube-aws-env/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM teliaoss/docker-sonarqube:7.3
```

Comment:
I'm assuming the above is a temporary thing until they release their own version? Perhaps you could submit a PR upstream to add the images?

Reply:
Sonarqube and (temporarily?) stopped providing docker files and official docker images on Dockerhub.  Forked their git repo (GPL3....) and added files for the new versions.  Hopefully just temporary.

Reply:
Roger!